src/main/java/org/broadinstitute/hellbender/engine/ReadlessAssemblyRegion.java

```diff
@@ -0,0 +1,58 @@
+package org.broadinstitute.hellbender.engine;
+
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+
+import java.util.Objects;
+
+/**
+ * A cut-down version of {@link AssemblyRegion} that doesn't store reads, used in the strict implementation of
+ * {@link org.broadinstitute.hellbender.engine.spark.FindAssemblyRegionsSpark}.
+ */
+public class ReadlessAssemblyRegion extends ShardBoundary {
```

Comment:
There should probably be a common interface linking this to AssemblyRegion, or a different name distinguishing it.

Reply:
Agreed.